tests/utils/concat-tests-test.js

```diff
@@ -11,13 +11,14 @@ var readContent   = require('../helpers/file');
 var getES6Package = require('../../lib/get-es6-package');
 var concatES6     = require('../../lib/utils/concat-es6');
 
-var fixtureLibPath  = path.join(__dirname, 'fixtures/packages/ember-metal/lib');
-var fixtureTestPath = path.join(__dirname, 'fixtures/packages/ember-metal/tests');
+var fixtureLibPath  = path.join(__dirname, '../fixtures/packages/ember-metal/lib');
+var fixtureTestPath = path.join(__dirname, '../fixtures/packages/ember-metal/tests');
```

Comment:
@rwjblue @twokul this seems to have done the trick.

The test couldn't find the fixture packages because it is in `tests/utils`.